### ISSUE TITLE
Blur is using a inexistent "enabled" attribute

### ISSUE THREAD

**Issue by chazgps** [gskinner/flutter_animate#41]
flutter_animated version 2.0.1 has Blur effect with a bug.

In blur_effect.dart ImageFiltered is using a inexistent "enabled" attribute on BlurEffect class.

I downgraded Dart SDK from 2.17.1 to 2.15.1 and not resolved.

![blur_error](https://user-images.githubusercontent.com/6288299/204505166-93a3f43b-5ecc-44ad-8514-bb424f12309c.png)

> flutter doctor
Doctor summary (to see all details, run flutter doctor -v):
[√] Flutter (Channel stable, 3.0.1, on Microsoft Windows [versÆo 10.0.22621.819], locale pt-BR)
[√] Android toolchain - develop for Android devices (Android SDK version 33.0.0-rc1)
[√] Chrome - develop for the web
[√] Visual Studio - develop for Windows (Visual Studio Community 2022 17.2.3)
[√] Android Studio (version 2021.1)
[√] Connected device (4 available)
[√] HTTP Host Availability

• No issues found!

> dart --version
Dart SDK version: 2.15.1 (stable) (Tue Dec 14 13:32:21 2021 +0100) on "windows_x64"

**Comment by gskinner** [comment]
I'm out of the office at the moment, so I haven't checked when it was added, but `enabled` is definitely a param:
https://api.flutter.dev/flutter/widgets/ImageFiltered/enabled.html

**Comment by chazgps** [comment]
OK, this parameter exists since the Apr 25 commit:
https://github.com/flutter/flutter/pull/102085

And belongs to 3.3.0 release, so, to use this class, I need to update my Flutter SDK to 3.3.0 version ? Really ?
https://docs.flutter.dev/development/tools/sdk/release-notes/release-notes-3.3.0

Can you substitute 'enabled' for a 'Visibity' widget for compability with older SDKs ?
Thanks!

**Comment by esDotDev** [comment]
I wonder if there is some sort of conditional compilation we could do here. In Unity we would just have:
```c#
#if UNITY_2022
  return ImageFiltered(enabled: ...);
#else 
  return ImageFiltered( ...);
#endif
```
Is there some dart/flutter equivalent? 

I know we could probably create some sort of shim class, and provide 2 different implementations of it using conditional imports. But that seems like a lot of work compared to the above.  

Hmm.. looks like we're not the only ones who can use this: https://github.com/dart-lang/sdk/issues/33249

**Comment by esDotDev** [comment]
@gskinner the most prudent thing to do on this for now might be to just comment this out, and then re-enable the property once this code reaches stable?

**Comment by chazgps** [comment]
both suggestions seem good to me !

**Comment by gskinner** [comment]
Okay. I apologize for the see-sawing on this, but after a lot of internal discussion I think I've reversed my decision. Having a policy that suggests the library will always be compatible with versions of Flutter going back 6 months introduces a huge testing burden I'm not eager to take on.

I know that with a young library that can be a bit of a pain, since new (hopefully useful) features are being added, but as it stabilizes this will be less of a problem, since devs can just stick to an older version until they are ready to update to the latest stable version of Flutter.

Closing this, but feedback is welcome.
